### PR TITLE
fix(transition) EVAL-158 fix before transition to set grant privileges on new schemas (viesco/notes) after cloning.

### PR DESCRIPTION
**Pull request by qpic-cgi**
## Describe your changes
Ajout d'un script permettant de donner les différentes autorisations pour les schemas clonés lors du before transition

## Checklist tests
Tester que le flow de transition passe.

## Issue ticket number and link
[Jira - EVAL-158](https://jira.support-ent.fr/browse/EVAL-158)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/competences/controllers/YearTransitionController.java [hunk 1]
```diff
@@ -55,16 +56,16 @@ public void beforeTransition(final HttpServerRequest request) {
     @ApiDoc("processing after transition and alimentation")
     @SecuredAction(value = "", type = ActionType.RESOURCE)
     @ResourceFilter(SuperAdminFilter.class)
-    public void afterTransition(final HttpServerRequest request){
```
Comment (qpic-cgi) [inline src/main/java/fr/openent/competences/controllers/YearTransitionController.java:58]: start indentation

File: src/main/java/fr/openent/competences/controllers/YearTransitionController.java [hunk 2]
```diff
@@ -55,16 +56,16 @@ public void beforeTransition(final HttpServerRequest request) {
...
+    public void afterTransition(final HttpServerRequest request) {
         log.info("Start transition after ...");
         List<Future> futures = new ArrayList<>();
         Future<JsonArray> future1 = Future.future();
         log.info("START clearTablePostTransition ...");
-        transitionService.clearTablePostTransition(event ->{
-            if(event.isRight()){
+        transitionService.clearTablePostTransition(event -> {
+            if (event.isRight()) {
```
Comment (qpic-cgi) [inline src/main/java/fr/openent/competences/controllers/YearTransitionController.java:65]: indentation jusqu'à la fin du fichier

File: src/main/java/fr/openent/competences/service/impl/CompetencesTransitionWorker.java [hunk 3]
```diff
@@ -21,7 +23,9 @@ public class CompetencesTransitionWorker extends BusModBase implements Handler<M
     @Override
     public void start() {
         super.start();
-        this.transitionService = new DefaultTransitionService();
+        Sql sqlAdmin = Sql.createInstance(vertx.eventBus(), config.getJsonObject(Field.POSTGRESCONFIG,
+                new JsonObject()).getString(Field.SQLADMINADRESS, Field.SQLPERSISTORADMIN));
```
Comment (BastienBenard) [inline src/main/java/fr/openent/competences/service/impl/CompetencesTransitionWorker.java:27]: Converti ta config en model, pou éviter la duplication de code.

File: src/main/java/fr/openent/competences/service/impl/DefaultTransitionService.java
Comment (BastienBenard) [inline src/main/java/fr/openent/competences/service/impl/DefaultTransitionService.java:1]: Faire un test pour ce service me parait pas mal

**Review by BastienBenard — APPROVED** [review]

**Pull request merged**